### PR TITLE
Re-enable arm64 ngen

### PR DESCRIPTION
**Pull request by MarcoRossignoli**
Re-enable arm64 ngen

### CODE REVIEW

**Comment by MarcoRossignoli** [comment]
@ankitvarmait I'm re-enabling arm64 ngen for test host using the preview, now manifest looks good for arm64 modules:
image

@davkean pls let us know if you see something strange inside logs.
@ankitvarmait some ETA for the rtm release?

**Review by fhnaseer — APPROVED** [review]

File: src/package/VSIXProject/TestPlatform.csproj [hunk 1]
```diff
@@ -252,12 +252,11 @@
...
+    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.arm64.exe" Ngen="true" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.arm64.exe" />
+    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net47.arm64.exe" Ngen="true" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net47.arm64.exe" />
+    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net471.arm64.exe" Ngen="true" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net471.arm64.exe" />
+    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net472.arm64.exe" Ngen="true" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net472.arm64.exe" />
+    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net48.arm64.exe" Ngen="true" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net48.arm64.exe" />
```
Comment (davkean) [inline src/package/VSIXProject/TestPlatform.csproj:259]: As a matter of interest why do you have architecture + target framework versions of your test host? 

I would consider targeting the lowest version and then set the https://docs.microsoft.com/en-us/dotnet/api/system.appdomainsetup.targetframeworkname?view=net-6.0 to the user's target to enable quirks for that executable.

Comment (MarcoRossignoli) [reply src/package/VSIXProject/TestPlatform.csproj:259]: I suppose historical reason but maybe @nohwnd has got better reason.

Comment (nohwnd) [reply src/package/VSIXProject/TestPlatform.csproj:259]: Reason is that I did not know you can do it this way when I added the additional testhosts, and no-one suggested it before we do it this way.

**Pull request merged**